<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keyOptionBlock</class>
 <widget class="QWidget" name="keyOptionBlock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="keyLabel">
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>说明</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="keyLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="resetBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>重设</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
